redaxo/src/core/fragments/core/Component/Dialog.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+use Redaxo\Core\Fragment\Component\Dialog;
+use Redaxo\Core\Fragment\Fragment;
+
+/** @var Dialog $this */
+?>
+<sl-dialog <?= $this->attributes->with([
+    'label' => $this->label,
```

Comment:
psalm meckert hier zurecht, dass die Property auch `Fragment` sein kann.
Falls es nur `string|null` sein soll, in der Klasse entsprechend ändern.